.scripts/common.ts

```diff
@@ -108,7 +108,7 @@ function isPackageFolderPath(folderPath: string, packagesToIgnore: string[]): bo
   if (fileExistsSync(packageJsonFilePath)) {
     const packageJson: PackageJson = readPackageJsonFileSync(packageJsonFilePath);
     // Skip all perf framework projects from gulp pack
-    if (packageJson.name.startsWith("@azure-tests/")) {
+    if (packageJson?.name?.startsWith("@azure-tests/")) {
```

Comment:
Can you also confirm if the failed pipeline run that you saw is fixed with this change?

Reply:
it succeed: https://dev.azure.com/azure-sdk/public/_build/results?buildId=938228&view=logs&j=ca395085-040a-526b-2ce8-bdc85f692774&t=2997d1dd-cd01-525b-c0a8-6ec662309496